site/content/contribute/getting-started/contribution-checklist.md

```diff
@@ -0,0 +1,30 @@
+---
+title: "Contribution Checklist"
+date: 2017-08-20T12:33:36-04:00
+weight: 1
+subsection: Getting Started
+---
+
+# Contribution Checklist
+
+Thanks for your interest in contributing to Mattermost! Come [join our "Contributors" community channel](https://pre-release.mattermost.com/core/channels/tickets) on our daily build server, where you can discuss questions with community members and the Mattermost core team.
+
+To help with translations, [see the localization process](https://docs.mattermost.com/developer/localization.html), follow this checklist for submitting a pull request (PR):
```

Comment:
The first part of the sentence is about translations and the second part is about submitting a PR, seems a bit split-brained. Maybe divide them into two sentences and put them on different lines?

Reply:
Yeah, I'll fix that. It's confusing as-is.